src/cli/rdb-cli.c

```diff
@@ -60,7 +60,7 @@ static void printUsage() {
     printf("\t-l, --log-file <PATH>         Path to the log file (Default: './rdb-cli.log')\n\n");
 
     printf("FORMAT_OPTIONS ('json'):\n");
-    printf("\t-w, --with-aux-values         Include auxiliary values\n");
+    printf("\t-i, --include <EXTRAS>        To include: {aux-val|func}\n");
```

Comment:
not sure these should be combined in the same option.
is this just for JSON?
i.e. for RESP, the functions are included by default?
in the json we format them in a readable manner, right?

Reply:
It is just for json part. RESP won't be affected. 
Generally speaking the usage or `rdb-cli` is constructed from two parts:
1. Configuring core parser and common features.
2. Configuring the FORMATTER that will be used.
```
Usage: rdb-cli /path/to/dump.rdb [OPTIONS] {json|resp|redis} [FORMAT_OPTIONS]
```

Reply:
Escaping is being applied over the entire json output.